DOCS.md

```diff
@@ -45,11 +44,11 @@ Sample records:
 
 Sample records:
 ```
- Row | decision_id | decision |              metrics             |   experiment_id   | mdp_id |     ts     |
------|-------------|----------|----------------------------------|-------------------|--------|------------|
-  1  |   c2aa520f  |  female  |        {"height": 158.462}       | height-prediction |    a   | 1585613418 |
-  2  |   f3e637a5  |   male   |        {"height": 172.331}       | height-prediction |    b   | 1585934016 |
-  3  |             |          | {"female": {"delayedMetric": 1}} | height-prediction |    a   | 1599999016 |
+ | Row | decision_id | decision | metrics                          | experiment_id     | mdp_id | ts         |
+ | --- | ----------- | -------- | -------------------------------- | ----------------- | ------ | ---------- |
+ | 1   | c2aa520f    | female   | {"height": 158.462}              | height-prediction | a      | 1585613418 |
+ | 2   | f3e637a5    | male     | {"height": 172.331}              | height-prediction | b      | 1585934016 |
+ | 3   |             |          | {"female": {"delayedMetric": 1}} | height-prediction | a      | 1599999016 |
```

Comment:
`ocd++`

Reply:
LOL.  I have a markdown table formatter.  can you imagine editing this by hand jfc